packages/pyiceberg/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: pyiceberg
+  version: 0.6.0
+  top-level:
+    - hive_metastore
+    - pyiceberg
+    - fb303
```

Comment:
I think we can remove `hive_metastore` and `fb303` from top-level keys unless they are public modules that users use.

Reply:
I'm happy to remove these upfront and if we need them for any reason later we can move them. I think this could be some licensing setup that doesn't apply to Pyodide's uses.